src/components/Text/index.js

```diff
@@ -45,16 +49,16 @@ export default class Text extends Component {
 
       this.state = {
          editing: props.editing,
-         inputBody: props.body
+         inputValue: props.editorValue
```

Comment:
Should we just call this `editorValue` instead of `inputValue` to keep the language consistent?